keyboards/dz60/keymaps/jarred/config.h

```diff
@@ -0,0 +1,8 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+ 
+#include "../../config.h"
+ 
+#define PREVENT_STUCK_MODIFIERS
```

Comment:
This is actually the new default now. This isn't needed.  But you may need to update your fork to get the changes.